### PR TITLE
Add email notifications for Emburse transactions

### PR DESCRIPTION
**Pull request by tmb**
fixes #67

### CODE REVIEW

File: app/jobs/sync_emburse_transactions_job.rb [hunk 1]
```diff
@@ -72,4 +74,8 @@ def notify_admin(emburse_t)
     emburse_t.update!(notified_admin_at: Time.now)
   end
 
+  def notify_user(emburse_t)
+    return if !emburse_t.event
```
Comment (lachlanjc) [inline app/jobs/sync_emburse_transactions_job.rb:78]:
```suggestion
    return unless emburse_t.event
```

File: app/mailers/emburse_transactions_mailer.rb [hunk 2]
```diff
@@ -5,4 +5,13 @@ def notify(params)
...
       subject: "[Action Requested] No department linked to Emburse transaction ##{@emburse_transaction.id}"
   end
+
+  def organizer_notify(params)
+    @event = params[:event]
+    @emburse_transaction = params[:emburse_transaction]
+
+    emails = @event.users.map { |u| u.email }
+    mail to: emails,
+      subject: "#{@emburse_transaction.card.user.name} spent #{@emburse_transaction.amount} at #{@emburse_transaction.merchant_name}"
```
Comment (lachlanjc) [inline app/mailers/emburse_transactions_mailer.rb]:
```suggestion
      subject: "#{@emburse_transaction.amount} spent at #{@emburse_transaction.merchant_name} by #{@emburse_transaction.card.user.name}"
```
Would prefer to have amount/venue before person

File: app/views/emburse_transactions_mailer/organizer_notify.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<p>Hi there,</p>
+
+<p>
+  <%= @emburse_transaction.card.user.name %>’s card was just used to spend <%= @emburse_transaction.amount %> at <%= @emburse_transaction.merchant_name %>.
```
Comment (lachlanjc) [inline app/views/emburse_transactions_mailer/organizer_notify.html.erb]: I'm not positive, but shouldn't `render_money` be used for the amount? Just making sure this is formatted correctly

**Review by lachlanjc — APPROVED** [review]
As long as everything works, gets my :+1:!

**Comment by maxwofford** [comment]
@tmb LGTM as long as it worked locally

**Pull request merged**